components/Accordion/src/accordion.tsx

```diff
@@ -32,7 +39,9 @@ export const Accordion: React.FC<AccordionProps> = ({ items, id }) => {
             aria-labelledby={item.id}
             data-bs-parent={`#${item.id}Accordion`}
           >
-            <div className="accordion-body">{item.render()}</div>
+            <div className="accordion-body" style={{ backgroundColor: item.backgroundColor }}>
+              {item.render()}
+            </div>
```

Comment:
Twee dingen om even over na te denken:

1. Eigenlijk willen we de styling in aparte bestanden hebben (in dit geval `accordion.css`);
2. Wat gebeurt er nu wanneer iemand geen `backgroundColor` heeft ingevuld? Wordt dan de `css` regel `background-color: undefined` naar de DOM gestuurd?

Reply:
Style zal ik volgende keer via een bestand doen en wat er naar de DOM gestuurd wordt zal ik zo even controleren